src/main/scala/util/package.scala

```diff
@@ -234,23 +234,55 @@ package object util {
     map.view.map({ case (k, vs) => k -> vs.toList }).toList
   }
 
+/** provides operators useful for working with bidirectional [[Bundle]]s
+  * 
+  * In terms of [[Flipped]] with a producer 'p' and 'consumer' c:
+  * c :<= p // means drive all unflipped fields of 'c' from 'p' (e.g.: valid/bits)  
+  * c :=> p // means drive all flipped fields of 'c' from 'p' (e.g.: ready)  
+  * c :<> p // do both of the above
+  * p :<> c // do both of the above, but you'll probably get a Gender error later.
+  * 
+  * This utility class is needed because in [[chisel3]]:
+  * c := p // only works if there are no directions on fields.  
+  * c <> p // only works if one of those is an [[IO]] (not a [[Wire]]).
+  * 
+  * Compared with [[chisel3]] operators:
+  * c <> p   is an 'actual-direction'-inferred 'c :<> p' or 'p :<> c'
+  * c := p   is equivalent to 'c :<= p' + 'p :=> c'
```

Comment:
```suggestion
  * c := p   is equivalent to 'c :<= p' + 'c :=> p'
```

@terpstra (and/or @leikou01 )  can you double check-this statement? Given the first two lines in this it seems like this change is what it should be. Otherwise I don't really understand this sentence.

Reply:
just tested it. the latter is correct.
`* c := p   is equivalent to 'c :<= p' + 'p :=> c' or 'c:<>= p'

Consumer always on LHS.

Reply:
ok, i will resolve this and leave it as-is.

Reply:
sorry, even the one sentence from myself is not consistent. corrected it in the above.